environment/src/main/java/com/sequenceiq/environment/network/dao/domain/BaseNetwork.java

```diff
@@ -266,6 +276,7 @@ public String toString() {
                 ", outboundInternetTraffic=" + outboundInternetTraffic +
                 ", accountId='" + accountId + '\'' +
                 ", resourceCrn='" + resourceCrn + '\'' +
+                ", privateDnsZoneId='" + privateDnsZoneId + '\'' +
```

Comment:
I don't think having a cloud specific parameter in the toString is a good idea for the `BaseNetwork` which presents for all the providers...

Reply:
Yes, this was an error, I deleted it.